README.md

```diff
@@ -27,3 +27,24 @@ Like:
             "arn:aws:iam::<AWS-ACCOUNT-ID>:mfa/${aws:username}"
         ]
     }
+
+### Installation
+
+- Install golang if it is not already on your machine. See: https://golang.org/doc/install
+
+- Set GOPATH environment variable if it does not exist. Ex `export GOPATH=~/dev/go`
+
+- Clone repository:
```

Comment:
Thanks for the PR, I guess no need to clone the repo: 

    $ go get github.com:boynux/aws-authenticator

This will do the job, would you please update that?